File: ClientCode/Assets/Scripts/Country.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Country //Class containing all info about Country
{
    string _countryName;
    int _newCases;
    int _allCases;
    int _newDeaths;
    int _allDeaths;
    int _Xcords;
    int _Ycords;


    //Constructors
    public Country()
    {
        _countryName = "";
        _newCases = 0;
        _allCases = 0;
        _newDeaths = 0;
        _allDeaths = 0;
        _Xcords = 0;
        _Ycords = 0;
    }

    public Country(string countryName, int newCases, int allCases, int newDeaths, int allDeaths)
    {
        _countryName = countryName;
        _newCases = newCases;
        _allCases = allCases;
        _newDeaths = newDeaths;
        _allDeaths = allDeaths;
    }

    public Country(string countryName, int newCases, int allCases, int newDeaths, int allDeaths, int XCords, int YCords)
    {
        _countryName = countryName;
        _newCases = newCases;
        _allCases = allCases;
        _newDeaths = newDeaths;
        _allDeaths = allDeaths;
        _Xcords = XCords;
        _Ycords = YCords;
    }

    //Getters and Setters
    public string GetName()
    {
        return _countryName;
    }
    public void SetName(string countryName)
    {
        _countryName = countryName;
    }
    public int GetNewCases()
    {
        return _newCases;
    }
    public void SetNewCases(int newCases)
    {
        _newCases = newCases;
    }

    public int GetAllCases()
    {
        return _allCases;
    }
    public void SetAllCases(int allCases)
    {
        _allCases = allCases;
    }
    public int GetNewDeaths()
    {
        return _newDeaths;
    }
    public void SetNewDeaths(int newDeaths)
    {
        _newDeaths = newDeaths;
    }

    public int GetAllDeaths()
    {
        return _allDeaths;
    }
    public void SetAllDeaths (int allDeaths)
    {
        _allDeaths = allDeaths;
    }
    public int GetX()
    {
        return _Xcords;
    }
    public int GetY()
    {
        return _Ycords;
    }
}

File: ServerAPI/Country.php
<?php
class Country
{
    public $CountryName;
    public $NewCases;
    public $TotalCases;
    public $NewDeaths;
    public $TotalDeaths;
    
    //County function constructor
    public function __construct($countryName, $newCases, $totalCases, $newDeaths, $totalDeaths)
    {
        $this->CountryName = $countryName;
        $this->NewCases = $newCases;
        $this->TotalCases = $totalCases;
        $this->NewDeaths = $newDeaths;
        $this->TotalDeaths = $totalDeaths;
    }

       //All getters
    public function getCountryName()
    {
        return $this->CountryName;
    }

    public function getNewCases()
    {
        return $this->NewCases;
    }

    public function getTotalCases()
    {
        return $this->TotalCases;
    }

    public function getNewDeaths()
    {
        return $this->NewDeaths;
    }

    public function getTotalDeaths()
    {
        return $this->TotalDeaths;
    }
}
?>